app/main/views/marketplace.py

```diff
@@ -288,6 +288,10 @@ def list_opportunities(framework_family):
         framework_family=framework['framework'],
         framework_family_name='Digital Outcomes and Specialists',
         lot_names=tuple(lot['name'] for lot in lots_by_slug.values() if lot['allowsBrief']),
+        outcome={
+            'closed': 'awaiting outcome',
+            'unsuccessful': 'no suitable suppliers'
```

Comment:
I know it's a bit verbose to have the `awarded` and `cancelled` ones here too, but if we want to change the content in the future it might be good to have all the strings in one place? Then we can just use `outcome[brief.status]` in the template, without the `.get`.

Reply:
Better yet, we could set the content dict in the `_briefs_results.html` template rather than the view 😸